python/lsst/ts/genericcamera/genericcameracsc.py

```diff
@@ -19,17 +19,20 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-__all__ = ["GenericCameraCsc", "run_genericcamera"]
+__all__ = ["GenericCameraCsc", "requests", "run_genericcamera"]
```

Comment:
Is this right? Is your idea to export the `requests` library imported below? Is there a particular reason for doing it?

Reply:
Yes, unfortunately the mocks in the unit test don't work because `requests` doesn't appear in the namespace. I tried all sorts of tricks to make it happen without doing this, but nothing worked. If you have suggestions, I'm all ears.